keps/prod-readiness/sig-storage/1979.yaml

```diff
@@ -0,0 +1,3 @@
+kep-number: 1979
+stable:
+  approver: "@xing-yang"
```

Comment:
PRR reviewer should be @deads2k or @wojtek-t or @johnbelamaric

Reply:
Validation will kick in once the kep.yaml is updated. Please update the metadata there and assign me for PRR approval.

Reply:
Will do it shortly